unstructured/partition/__init__.py

```diff
@@ -1,17 +1,16 @@
 import requests  # type: ignore
-from typing import BinaryIO, List, Optional, Union, Tuple, Mapping
+from typing import BinaryIO, Optional, Union, Tuple, Mapping
 from urllib.parse import urlsplit
 
-from unstructured.documents.elements import Element
-
 
 def _partition_via_api(
     filename: str = "",
     file: Optional[bytes] = None,
     url: str = "https://ml.unstructured.io/layout/pdf",
     token: Optional[str] = None,
     data: Optional[dict] = None,  # NOTE(alan): Remove after different models are handled by routing
-) -> List[Element]:
```

Comment:
Shouldn't we keep the return type hint?

Reply:
The type hint for this one was actually wrong (it returns a list of dicts), but when I updated it `mypy` had a meltdown. I'll take another look and see if I can fix it.